autograd/numpy/fft.py

```diff
@@ -1,4 +1,5 @@
 from __future__ import absolute_import
+import numpy as np
```

Comment:
In this case, numpy is used to generate a factor array of the same shape as the input/output. These functions are never taken gradient of.